WrightTools/data/_join.py

```diff
@@ -88,6 +88,19 @@ def join(
     for d in datas[1:]:
         if d.axis_expressions != axis_expressions:
             raise wt_exceptions.ValueError("Joined data must have same axis_expressions")
+        for a in d.axes:
+            if len(np.squeeze(a.variables[0][:]).shape) > 1:
```

Comment:
consider the ndim attribute (for clarity):  `a.variables[0][:].squeeze().ndim > 1`